database.py

```
@@ -80,6 +80,12 @@ def _create_backup(self, file_path: Path) -> bool:
                 return False
         return False
 
+    def _unstable_helper(self, items: list = []):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)